Setup fails with error compiling clap in macOS and Ubuntu

## Expected Behavior
`setup.sh` run successfully.

## Actual Behavior
`setup.sh` doesn't complete due to error while compiling clap.

## Steps to Reproduce the Problem

1. Execute `./setup.sh`
Produces the following output:

```bash
[-] Copying additional boards to Tock... DONE.
[-] Applying patch "01-persistent-storage.patch"... DONE.
[-] Applying patch "02-usb.patch"... DONE.
[-] Applying patch "03-usb-debugging.patch"... DONE.
[-] Applying patch "04-additional-boards.patch"... DONE.
[-] Applying patch "05-mpu-fix.patch"... DONE.
[-] Applying patch "06-update-uicr.patch"... DONE.
[-] Applying patch "07-firmware-protect.patch"... DONE.
Signature ok
subject=/CN=OpenSK CA
Getting Private key
Signature ok
subject=/CN=OpenSK Hacker Edition
Getting CA Private Key
info: syncing channel updates for 'nightly-2020-06-10-x86_64-apple-darwin'

  nightly-2020-06-10-x86_64-apple-darwin unchanged - rustc 1.46.0-nightly (feb3536eb 2020-06-09)

info: checking for self-updates
Requirement already satisfied: tockloader==1.5 in /Library/Python/3.8/site-packages (1.5.0)
Requirement already satisfied: six in /Users/dds/Library/Python/3.8/lib/python/site-packages (1.16.0)
Requirement already satisfied: intelhex in /Library/Python/3.8/site-packages (2.3.0)
Requirement already satisfied: pytoml>=0.1.11 in /Library/Python/3.8/site-packages (from tockloader==1.5) (0.1.21)
Requirement already satisfied: colorama>=0.3.7 in /Library/Python/3.8/site-packages (from tockloader==1.5) (0.4.4)
Requirement already satisfied: pyserial>=3.0.1 in /Library/Python/3.8/site-packages (from tockloader==1.5) (3.5)
Requirement already satisfied: argcomplete>=1.8.2 in /Library/Python/3.8/site-packages (from tockloader==1.5) (1.12.3)
Requirement already satisfied: crcmod>=1.7 in /Library/Python/3.8/site-packages (from tockloader==1.5) (1.7)
info: component 'rust-std' for target 'thumbv7em-none-eabi' is up to date
    Updating crates.io index
  Installing elf2tab v0.6.0
   Compiling libc v0.2.99
   Compiling proc-macro2 v1.0.28
   Compiling version_check v0.9.3
   Compiling autocfg v1.0.1
   Compiling unicode-xid v0.2.2
   Compiling syn v1.0.74
   Compiling unicode-segmentation v1.8.0
   Compiling unicode-width v0.1.8
   Compiling ansi_term v0.11.0
   Compiling cfg-if v1.0.0
   Compiling bitflags v1.3.1
   Compiling vec_map v0.8.2
   Compiling strsim v0.8.0
   Compiling byteorder v0.5.3
   Compiling lazy_static v1.4.0
   Compiling heck v0.3.3
   Compiling textwrap v0.11.0
   Compiling num-traits v0.2.14
   Compiling num-integer v0.1.44
   Compiling proc-macro-error-attr v1.0.4
   Compiling proc-macro-error v1.0.4
   Compiling elf v0.0.10
   Compiling quote v1.0.9
   Compiling atty v0.2.14
   Compiling time v0.1.44
   Compiling filetime v0.2.15
   Compiling xattr v0.2.2
   Compiling clap v2.33.3
error[E0658]: `if` is not allowed in a `const fn`
  --> /Users/david/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.33.3/src/app/settings.rs:7:1
   |
7  | / bitflags! {
8  | |     struct Flags: u64 {
9  | |         const SC_NEGATE_REQS       = 1;
10 | |         const SC_REQUIRED          = 1 << 1;
...  |
51 | |     }
52 | | }
   | |_^
   |
   = note: see issue #49146 <https://github.com/rust-lang/rust/issues/49146> for more information
   = help: add `#![feature(const_if_match)]` to the crate attributes to enable
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0658]: `if` is not allowed in a `const fn`
  --> /Users/david/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.33.3/src/args/settings.rs:6:1
   |
6  | / bitflags! {
7  | |     struct Flags: u32 {
8  | |         const REQUIRED         = 1;
9  | |         const MULTIPLE         = 1 << 1;
...  |
28 | |     }
29 | | }
   | |_^
   |
   = note: see issue #49146 <https://github.com/rust-lang/rust/issues/49146> for more information
   = help: add `#![feature(const_if_match)]` to the crate attributes to enable
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

   Compiling tar v0.4.37
   Compiling chrono v0.4.19
error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
error: could not compile `clap`.

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `elf2tab v0.6.0`, intermediate artifacts can be found at `/var/folders/sc/c77xmhjn1rx5gffjzxl2p_dm0000gn/T/cargo-installpItNMj`

Caused by:
  build failed
```

## Specifications

- Version: current stable (commit e02eaa2cacb6f2949fe282d9e32129db30dccfbb)
- Platform: nrf52840_dongle_dfu
- OS: macOS Catalina 10.15.7 and Ubuntu 20.04

--- a/fuzzing_setup.sh
+++ b/fuzzing_setup.sh
@@ -20,4 +20,4 @@ done_text="$(tput bold)DONE.$(tput sgr0)"
 set -e
 
 # Install cargo-fuzz library.
-cargo install cargo-fuzz
+cargo +stable install cargo-fuzz

--- a/setup.sh
+++ b/setup.sh
@@ -43,7 +43,8 @@ rustup target add thumbv7em-none-eabi
 
 # Install dependency to create applications.
 mkdir -p elf2tab
-cargo install elf2tab --version 0.6.0 --root elf2tab/
+rustup install stable
+cargo +stable install elf2tab --version 0.6.0 --root elf2tab/
 
 # Install python dependencies to factory configure OpenSK (crypto, JTAG lockdown)
 pip3 install --user --upgrade colorama tqdm cryptography "fido2>=0.9.1"

